style.css

```diff
@@ -129,3 +129,36 @@ main {
   border-width: 2px;
   padding: 5px;
 }
+
+footer {
+  margin-bottom: 10vh;
+}
+
+.navbar {
+  background-color: rgb(53, 110, 91);
+  width: 100%;
+  display: flex;
+  position: fixed;
+  bottom: 0;
+  justify-content: space-evenly;
+  z-index: 1;
+  border-radius: 15px 15px 0 0;
+  box-shadow: 0px 0px 10px 2px rgba(0, 0, 0, 0.5);
+  padding: 0;
+  /* Wie kann ich die Icons einfärben? */
```

Comment:
Icons einfärben geht direkt in der svg-Datei. Kann ich Dir gerne morgen noch mal zeigen, wenn ich es denn selbst hinbekomme. Meistens: im path ein fill-Attribut hinzufügen und dem die gewünschte Farbe zuweisen (ich hab's mit hex gemacht, das ging, also zB #ffffff).